src/rqt_plot/plot_widget.py

```diff
@@ -123,11 +123,15 @@ def get_plot_fields(node, topic_name):
         if is_array_or_sequence:
             if not has_index:
                 return [], base_error_msg + f'{name} is a nested type but not index provided'
```

Comment:
I know this is preexisting, but since we are touching this code:

```suggestion
                return [], base_error_msg + f'{name} is a nested type but no index provided'
```